src/control/jobs/control_jobs.c

```diff
@@ -848,12 +849,16 @@ static gboolean _dt_delete_dialog_main_thread(gpointer user_data)
 
   if (modal_dialog->send_to_trash)
   {
-    gtk_dialog_add_button(GTK_DIALOG(dialog), _("physically delete"), _DT_DELETE_DIALOG_CHOICE_DELETE);
```

Comment:
I would keep the "physically" here. This has been added because there was confusion between "delete" and "remove". From a user point of view the difference is not clear and made confusion. As a side effect, keep as much as possible the current strings is good (and a goal here) as this is planned for 3.8.1.

Reply:
I tried to keep the text in the buttons as short as possible because they are quite long already :stuck_out_tongue_closed_eyes:  Added 'physically' to dialog text and buttons.
Also moved a question mark into an earlier row.
If this is fine for you I will squash these commits.